.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+# Type of Issue:
```

Comment:
I see your rationale for adding this, but I'm not sure it's worth asking users to enter the type of issue: I don't feel it's fair to make standard users understand all of the different issue types, and if they don't understand them all (and/or we don't list all of the possible values), then there's a high likelihood that they'll enter the wrong type. i.e., it seems like this puts extra burden on the issue submitter without providing us with a reliable benefit.